src/core/lv_obj_property.h

```diff
@@ -107,14 +129,25 @@ lv_result_t lv_obj_set_properties(lv_obj_t * obj, const lv_property_t * value, u
  *====================*/
 
 /**
- * Read property value from object
+ * Read property value from object.
+ * If id is a style property, the style selector is default to 0.
  * @param obj       pointer to an object
  * @param id        ID of which property
  * @param value     pointer to a buffer to store the value
  * @return ? to be discussed, LV_RESULT_OK or LV_RESULT_INVALID
  */
 lv_property_t lv_obj_get_property(lv_obj_t * obj, lv_prop_id_t id);
 
+/**
+ * Read property value from object
+ * @param obj       pointer to an object
+ * @param id        ID of which property
+ * @param value     pointer to a buffer to store the value
+ * @param selector  selector for the style property. Ignored for non-style property.
+ * @return ? to be discussed, LV_RESULT_OK or LV_RESULT_INVALID
+ */
+lv_property_t lv_obj_get_style_property(lv_obj_t * obj, lv_prop_id_t id, uint32_t selector);
```

Comment:
Can these doc comments be improved any more? i.e., remove the `value` param, update the return value, make the description different for both functions.

Reply:
Thank you for pointing it out. I have corrected them. Please help to review again.